man/IRkernel.Rd

```diff
@@ -9,9 +9,7 @@
 \title{An R kernel for Jupyter.}
 \format{An object of class \code{list} of length 5.}
 \usage{
-options(jupyter.* = ...)
-getOption('jupyter.*')
-jupyter_option_defaults$jupyter.*
+jupyter_option_defaults
```

Comment:
always sad when automated checks make things actually *worse*.

we should maybe just pick one option that we set and get as an example instead of removing the examples.